Dependencies.props

```diff
@@ -9,12 +9,13 @@
     <Microsoft_SourceLink_Vsts_Git_Version>1.0.0-beta2-19367-01</Microsoft_SourceLink_Vsts_Git_Version>
     <MSTest_TestAdapter_Version>2.0.0</MSTest_TestAdapter_Version>
     <MSTest_TestFramework_Version>2.0.0</MSTest_TestFramework_Version>
+    <System_Configuration_ConfigurationManager_Version>4.6.0</System_Configuration_ConfigurationManager_Version>
```

Comment:
Just to confirm - moving this from Pinned to AutoUpdate was deliberate?

Reply:
Yes, currently there's no need for it not to be autoupdated.

Reply:
Great that we can move this to autoupdate :smile: